Makefile

```diff
@@ -60,10 +60,12 @@ clean: ## Make a clean source tree
 	-make -C docs clean
 	-make -C etc clean
 
+clean-env: clean remove-env  ## Make clean + remove conda env
```

Comment:
I prefer we not introduce another "env" target when this could be done with a simple command already. I think the times we need to clean up both the env and the build in a single command will be rare and `clean-env` and `remove-env` are too similar - which leads to confusion.